src/components/update-business-types.js

```diff
@@ -1,39 +1,84 @@
 const fs = require("fs");
 const fetch = require("node-fetch");
 
+// Mapping V2 Business Type strings to V1 strings
+const v1BusinessTypesMapping = {
+  "Hunter and trapper": "Hunting and trapping",
```

Comment:
there is a corresponding file to this one in validation repo - should we update/delete that one?

Reply:
Not exactly, the validation repo just has the v2 keys/strings for the actual enum.

Reply:
ah yes, it doesn't need the v1. thanks